fabric/src/main/java/com/wynntils/fabric/mixins/RenderTargetMixin.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright © Wynntils 2022.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.fabric.mixins;
+
+import com.mojang.blaze3d.pipeline.RenderTarget;
+import org.lwjgl.opengl.GL30;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.ModifyArgs;
+import org.spongepowered.asm.mixin.injection.Slice;
+import org.spongepowered.asm.mixin.injection.invoke.arg.Args;
+
+/*
+ * See https://gist.github.com/burgerguy/8233170683ad93eea6aa27ee02a5c4d1
+ */
+@Mixin(RenderTarget.class)
+public class RenderTargetMixin {
+
+    @ModifyArgs(
+            method = "createBuffers",
+            at =
+                    @At(
+                            value = "INVOKE",
+                            target =
+                                    "Lcom/mojang/blaze3d/platform/GlStateManager;_texImage2D(IIIIIIIILjava/nio/IntBuffer;)V",
+                            ordinal = 0,
+                            remap = false))
+    public void init(Args args) {
+        args.set(2, GL30.GL_DEPTH32F_STENCIL8);
```

Comment:
What will happen with this if we're not on Wynncraft? Can it interact badly with other mods, or plain Vanilla behavior elsewhere?

Reply:
This is identical to forge/burger's impl, and I don't know enough about rendering to see if it will work when returning not on server